apps/web/src/components/app/Player.test.tsx

```
@@ -0,0 +1,114 @@
+  PlayerActions: () => <div data-testid="player-actions" />,
+}));
+vi.mock('./player/PlayerControls', () => ({
+  PlayerControls: () => <div data-testid="player-controls" />,
+}));
+vi.mock('./player/PlayerMobile', () => ({
+  PlayerMobile: () => <div data-testid="player-mobile" />,
+}));
+vi.mock('./player/PlayerTrackInfo', () => ({
+  PlayerTrackInfo: ({ onSeek }: { onSeek: (time: number) => void }) => (
+    <div data-testid="player-track-info">
+      <button onClick={() => onSeek(42)}>Seek</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)